.github/workflows/helm-gke-deploy.yaml

```diff
@@ -70,11 +71,14 @@ on:
         description: "GKE cluster for authentication"
         required: true
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-helm-deploy
```

Comment:
For the Kustomize you included the `gke` in the suffix, but for Helm you don't? Please be consistent about your suffixes

Reply:
my bas.. i missed those two. But the idea is to give the nave of the workflow